### ISSUE TITLE
False-positive for ngrx-avoid-dispatching-multiple-actions-sequentially in subscriptions

### ISSUE THREAD

**Issue by bopm** [timdeschryver/eslint-plugin-ngrx#86]
Having onNgInit of my component like this:

```
ngOnInit(): void {
    this.route.params.subscribe(({ id }) => {
      this.id = id;
      if (id) {
        this.store.dispatch(
          load({
            id: this.id,
            page: this.page,
            per_page: this.limit,
            ransack: this.conditions
          })
        );
      }
    });

    this.form
      .get('search')
      .valueChanges.pipe(
        debounceTime(1000),
        takeWhile(() => this.alive)
      )
      .subscribe(value => {
        if (value.length > 0) {
          this.conditions = {
            ...this.conditions,
            code_or_name_cont: value
          };
        } else {
          const { code_or_name_cont, ...conditions } = this.conditions;
          this.conditions = conditions;
        }
        this.store.dispatch(
          load({
            id: this.id,
            page: this.page,
            per_page: this.limit,
            ransack: this.conditions
          })
        );
      });
    this.store.pipe(select(getActiveId), take(1)).subscribe(id => {
      if (Number(this.id) !== id) {
        this.store.dispatch(changeActive({ id: id }));
      }
    });
  }
```
leads to false-positive ngrx-avoid-dispatching-multiple-actions-sequentially on second and third dispatch.

**Comment by bopm** [comment]
@timdeschryver I was suspecting that. Another reason to speed-up migration for me.
